src/main/resources/templates/jenkins/rust/regularRuns/pipeline.groovy

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file configures the actual build steps for the automatic grading.
+ *
+ * !!!
+ * For regular exercises, there is no need to make changes to this file.
+ * Only this base configuration is actively supported by the Artemis maintainers
+ * and/or your Artemis instance administrators.
+ * !!!
+ */
+
+dockerImage = '#dockerImage'
+dockerFlags = '#dockerArgs'
+
+/**
+ * Main function called by Jenkins.
+ */
+void testRunner() {
+    docker.image(dockerImage).inside(dockerFlags) { c ->
+        runTestSteps()
+    }
+}
+
+private void runTestSteps() {
+    test()
+}
+
+/**
+ * Run unit tests
+ */
+private void test() {
+    stage('Test') {
+        sh 'cargo nextest run --profile ci'
+    }
+}
+
+/**
+ * Script of the post build tasks aggregating all JUnit files in $WORKSPACE/results.
+ *
+ * Called by Jenkins.
+ */
+void postBuildTasks() {
+    sh '''
+    rm -rf results
+    mkdir results
+    if [ -e target/nextest/ci/junit.xml ]
+    then
+        sed -i 's/<testsuites[^>]*>//g ; s/<\\/testsuites>//g' target/nextest/ci/junit.xml
```

Comment:
```suggestion
        sed -i 's/<testsuites[^>]*>/<testsuite>/g ; s/<\\/testsuites>/<\\/testsuite>/g' target/nextest/ci/junit.xml
```

Reply:
I noticed this with the JavaScript template and added a convoluted workaround. Didn't know you could nest `<testsuite>`s. I forgot to add this here, thanks for noticing.